src/richie/plugins/lti_consumer/models.py

```diff
@@ -76,15 +95,38 @@ def lti_provider(self):
             self.lti_provider_id, {}
         )
 
+    @property
+    def inline_ratio(self):
```

Comment:
I find it weird to name model fields with an underscore and would say the usual pattern in Django for this is what you see in drf or class based views for example:
The field is called `inline_ratio` and you get it via a method called `get_inline_ratio`

Reply:
Ok ! I wonder what is the best practice about that. I choose the syntax with `_` prefix as those values dot not aims to be used directly. I will use getter methods.